.github/CODEOWNERS

```diff
@@ -69,5 +69,6 @@
 /trace/**/*                            @ymotongpoo @GoogleCloudPlatform/python-samples-reviewers
 /translate/**/*                        @nicain @GoogleCloudPlatform/python-samples-reviewers
 /talent/**/*                           @GoogleCloudPlatform/python-samples-reviewers
+/vision/**/*                           @googleapis/python-samples-reviewers @googleapis/cdpe-cloudai
```

Comment:
It needs to be a team in the `GoogleCloudPlatform` org. If you're not sure of the correct one, we can leave it defaulted to `python-sample-reviewers` for now.

Reply:
Ok, done.  That's what it was in the old repo, and I couldn't find the same team in the GoogleCloudPlatform org.